app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,43 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import kotlinx.android.synthetic.main.activity_home.*
+import javax.inject.Inject
+
+/**
+ * Application home;
+ */
+class HomeActivity : WolmoActivity() {
+
+    private lateinit var mSimpleFragmentPagerAdapter: SimpleFragmentPagerAdapter
```

Comment:
Move this attribute below @Inject.

Reply:
Reorganized elements, first injected fields then the others.